cmd/faFindFast/efficient.go

```diff
@@ -73,6 +73,10 @@ func speedyWindowDifference(windowSize int, reference []dna.Base, query []dna.Ba
 	var gapOpenCloseRef, gapOpenQuery, gapClosedQuery, numRefNs, numQueryNsGap, numQueryNsMatch, numSubst int // ints we will get back when moving the window one ref base
 	var err error
 	var percentDiverged, rawPValue float64
+	var scorePValueCache map[int]float64
```

Comment:
I think we could use some comments for this line to know what values the cache is mapping between, and also what a divergence rate of -1 means.  I think this function could also use a comment.